.travis.yml

```diff
@@ -83,9 +86,10 @@ script:
     - docker ps --all
     - docker images
     - docker rmi localhost:5000/my-alpine
+    - docker logout localhost:5000
     # Setting up  Docker Registry is complete, let's do Buildah testing! 
     - make install.tools install.libseccomp.sudo all runc validate SECURITYTAGS="apparmor seccomp"
-    - go test -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
+    - go test -coverprofile=coverage.txt -covermode=atomic -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
```

Comment:
I'm wondering why we dont execute the make targets from the makefile here (`make test-unit`)

Reply:
I went by the example from the codecov site, but I saw what you did with CRIO and may convert to the Makefile approach.  It certainly would be easier to control from Make that way.  But really may main focus on this patch was just to get numbers showing up at the CodeCov site, currently I'm only getting a blank file there.  That might be explained by one of your earlier comments.

Thanks so much for eyeballing this, I truly appreciate it!